pkg/cluster/k8sres.go

```diff
@@ -1169,10 +1198,25 @@ func (c *Cluster) generateCloneEnvironment(description *acidv1.CloneDescription)
 			})
 	} else {
 		// cloning with S3, find out the bucket to clone
+		cloneScope := cluster
+		cloneS3Bucket := c.OpConfig.WALES3Bucket
+		cloneSuffix := getBucketScopeSuffix(description.UID)
+
+		customVars := customCloneVars(description.S3WalPath)
+		// if the provided path seems valid, then override
+		if len(customVars) > 2 {
+			msg := "Use custom S3WalPath %s from the manifest"
+			c.logger.Warningf(msg, description.S3WalPath)
```

Comment:
I see an issue here, unfortunately. This would block migrations towards K8S from non matching S3 paths? Where e.g. we do not have the uuid?